test/integration/tasklist_page_test.rb

```diff
@@ -1,47 +1,57 @@
 require 'integration_test_helper'
 
 class TasklistPageTest < ActionDispatch::IntegrationTest
-  it "renders the learn to drive page" do
-    visit "/learn-to-drive-a-car"
+  it "renders the end a civil partnership tasklist page" do
```

Comment:
We're not getting rid of the learn to drive page, so we probably want to keep some tests around for it too.